src/import/ImportPCM.cpp

```diff
@@ -373,20 +385,16 @@ ProgressResult PCMImportFileHandle::Import(WaveTrackFactory *trackFactory,
             framescompleted += block;
          }
 
-         updateResult = mProgress->Update(
-            framescompleted.as_long_long(),
-            fileTotalFrames.as_long_long()
-         );
-         if (updateResult != ProgressResult::Success)
-            break;
-
-      } while (block > 0);
+         progressListener.OnImportProgress(framescompleted.as_double() / fileTotalFrames.as_double());
```

Comment:
And one more denominator.

Maybe `OnImportProgress` should take numerator and denominator as two arguments, and you can write the check for zero division in one place only.